tests/test-env-check.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# check for python3
+pytoutput=$(python3 --version 2>&1)
+if [[ ${pytoutput:0:8} != "Python 3" ]]
+then
+    pyoutput=$(python --version 2>&1)
+    if [[ ${pyoutput:0:8} != "Python 3" ]]
+    then
+        echo "Python 3 is not installed/enabled."
+        return
+    else
+        if [[ ${pyoutput:9:1} < 5 ]]
+        then
+            echo "Python 3 is too old. Need at least 3.5. "
+            return
+        else
+            echo "$pyoutput detected!"
+        fi
+    fi
+else
+    if [[ ${pytoutput:9:1} < 5 ]]
+    then
+        echo "Python 3 is too old. Need at least 3.5. "
+        return
+    else
+        echo "$pytoutput detected!"
+    fi
+fi
+
+# check for g++ -- for autest
```

Comment:
I'm not convinced we need this check on compilers, it's already done in configure.ac.

Reply:
I believe he did this check as on some system the install os psutils needed for autest does not have a precompile .whl file for the system. not having g++ will mean autest will fail to install